Makefile-rpm-ostree.am

```diff
@@ -74,6 +74,9 @@ rpm_ostree_LDADD = $(PKGDEP_RPMOSTREE_LIBS) librpmostreepriv.la librpmostree-1.l
 # https://github.com/ostreedev/ostree/commit/1f832597fc83fda6cb8daf48c4495a9e1590774c
 # https://github.com/rust-lang/rust/issues/47714
 rpm_ostree_LDADD += -ldl
+if RUST_DEBUG
+rpm_ostree_LDADD += -lm
```

Comment:
Otherwise I get:

```
/bin/ld: /home/jlebon/Code/github.com/projectatomic/rpm-ostree/target/debug/librpmostree_rust.a(rand-320efdfff4960a7d.rand13.rcgu.o): undefined reference to symbol 'exp@@GLIBC_2.2.5'
//usr/lib64/libm.so.6: error adding symbols: DSO missing from command line
```

I *think* this is because some `rand` tests make use of `exp()`.

Reply:
Let's just unconditionally link it?  It's not like `-lm` is something exotic, and I can imagine us gaining a `rand` dependency in release mode too.